spec/score_card_spec.rb

```diff
@@ -0,0 +1,130 @@
+# frozen_string_literal: true
+
+require 'score_card'
+
+describe ScoreCard do
+  let(:bowl) { ScoreCard.new }
+  let(:frame) { double :frame }
+  describe '#frames' do
+    it 'should be 1 by default' do
```

Comment:
I think this it statement could be clearer - maybe something like 'should exist', 'should be an empty array' / 'should be empty', 'should be empty before any frames have been completed'?